worlds/rabi_ribi/client/client.py

```diff
@@ -555,6 +561,10 @@ def is_on_main_menu(self):
     def open_warp_menu(self):
         # Reenable the Strange Box first.
         self.rr_interface.set_item_state(STRANGE_BOX_ITEM_ID, 1)
+        if(self.rr_interface.is_near_crosswarp() and self.is_crosswarp_disabled):
```

Comment:
Remove parentheses and list `self.is_crosswarp_disabled` first; there's no need to scan memory if the check is disabled.